frontend/src/App.tsx

```diff
@@ -57,6 +57,14 @@ const App: React.FunctionComponent = () => {
         }
       }
     });
+    // const resp = await fetch("https://unpkg.com/electron@8.1.0/electron.d.ts")
```

Comment:
this shows just another example how for example the Electron project typings are working.